ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -329,6 +329,12 @@ tests:
     cluster_profile: gcp
     workflow: openshift-e2e-gcp-upi-xpn
   timeout: 6h0m0s
```

Comment:
Install succeeded in the latest rehearsal! The rehearsal failed because the job timed out at 4 hours. This is due to the fact that installer build times have gotten out of control. As a temporary workaround until we can fix the build issue, can you add this same timeout for 6 hours to the xpn config? You will then need to regenerate the job with `make update`. At that point, I think we should be ready to merge!